.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+# Issue Title
+
+Be as descriptive as you can with your title. If your issue is requesting
+a new topic, please preface the title with [NewTopic].
+
+# General
+
+We use issues to drive the discussion for changes to
+existing topics, and the creation of new topics.
+
+This way, we have those discussions before significant
```

Comment:
This text will show up inside the Write box when you're creating an issue @Rick-Anderson. Similar to what docfx GH has: 
https://github.com/dotnet/docfx/issues/new

Adding links from the top just adds more text to that box. We should keep it to a minimal.